curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/664ed9344ba3495e1f67f8ea.md

```diff
@@ -0,0 +1,58 @@
+---
+id: 664ed9344ba3495e1f67f8ea
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+Now that you have reviewed declaration and initialization, remove the code you wrote for your review.
+
+Do not remove your `character` variable.
+
+# --hints--
+
+You should not have a variable called profession.
+
+```js
+assert.notMatch(code, /profession/);
+```
+
+You should not have a variable called age.
+
+```js
+assert.notMatch(code, /age/);
+```
+
+You should not have a console.log statement for age.
```

Comment:
As an aside, the two log tests will never run because of the test for the variable name before them.

So as it is now, if you leave the console log but remove the variable declaration, you will get the assert message for the variable and not the log assert.

We might want to update the tests to avoid any confusion, I'm not sure if it is a big deal, but it does make the log tests a bit pointless.

Reply:
I understand what you are trying to say here. I can make changes to how the test for having  an `age` variable is written to check for `let age` which will make these two test work fine assuming the camper has entered the correct code from the previous step. I feel like this implementation is fine due to this reason.